atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraVerifier.java

```diff
@@ -211,7 +211,8 @@ private static void attemptToCreateKeyspaceOnHost(InetSocketAddress host, Cassan
         CassandraKeyValueServices.waitForSchemaVersions(
                 client,
                 "(adding the initial empty keyspace)",
-                config.schemaMutationTimeoutMillis());
+                config.schemaMutationTimeoutMillis(),
+                true);
```

Comment:
This is needed to allow our ETE tests to run with 2 C* nodes up on a 3 node C* cluster. Partially fixes https://github.com/palantir/atlasdb/issues/1705.